test/docker/compose.go

```diff
@@ -571,8 +571,8 @@ func (d *Compose) Start(ctx context.Context) (*DockerCompose, error) {
 		}
 		containers = append(containers, container)
 	}
-	if d.withPaLMApiKey != "" {
-		envSettings["PALM_APIKEY"] = d.withPaLMApiKey
+	if d.withGoogleApiKey != "" {
+		envSettings["PALM_APIKEY"] = d.withGoogleApiKey
```

Comment:
wondering if `envSettings["GOOGLE_APIKEY"]` should be set instead... and with the value of legacy env variable if new one is not set?

Reply:
good catch! will change that!